dbt/adapters/dremio/api/rest/endpoints.py

```diff
@@ -100,8 +109,22 @@ def _check_error(r, details=""):
         raise DremioPermissionException("No permission:" + details, error, r)
     if code == 404:
         raise DremioNotFoundException("Not found:" + details, error, r)
+    if code == 408:
+        raise DremioRequestTimeoutException("Request timeout:" + details, error, r)
     if code == 409:
         raise DremioAlreadyExistsException("Already exists:" + details, error, r)
+    if code == 429:
+        raise DremioTooManyRequestsException("Too many requests:" + details, error, r)
+    if code == 500:
+        raise DremioInternalServerException(
+            "Internal server error:" + details, error, r
+        )
+    if code == 503:
+        raise DremioServiceUnavailableException(
+            "Too many requests:" + details, error, r
+        )
+    if code == 504:
+        raise DremioGatewayTimeoutException("Too many requests:" + details, error, r)
```

Comment:
Is "Too many requests" the correct error message?

Reply:
Oh whoops, I'll fix that.